Usiqee/Module/Account/NotLogged/NotLoggedView.swift

```diff
@@ -0,0 +1,80 @@
+//
+//  NotLoggedView.swift
+//  Usiqee
+//
+//  Created by Amine on 02/05/2021.
+//
+
+import UIKit
+
+protocol NotLoggedViewDelegate: class {
+    func showPreAuthentication()
+}
+
+class NotLoggedView: UIView {
+    
+    // MARK: - Constants
+    private enum Constants {
+        static let identifier = "NotLoggedView"
+    }
+    
+    // MARK: - IBOutlet
+    @IBOutlet weak private var contentView: UIView!
+    @IBOutlet weak private var titleLabel: UILabel!
+    @IBOutlet weak private var subtitleLabel: UILabel!
+    @IBOutlet weak private var continueButton: UIButton!
+    
+    // MARK: - Properties
+    weak var delegate: NotLoggedViewDelegate?
+    
+    // MARK: - Lifecycle
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        commonInit()
+    }
+    
+    required init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+        commonInit()
+    }
+    
+    // MARK: - Private
+    private func commonInit() {
+        loadView()
+        setupView()
+    }
+    
+    private func loadView() {
+        Bundle.main.loadNibNamed(Constants.identifier, owner: self, options: nil)
+        addSubview(contentView)
+        contentView.frame = self.bounds
+        contentView.autoresizingMask = [.flexibleHeight, .flexibleWidth]
+    }
+    
+    private func setupView() {
+        setupLabels()
+        setupContinueButton()
+    }
+    
+    private func setupContinueButton() {
+        continueButton.setTitle(L10N.AccountNotLogged.continue, for: .normal)
+        continueButton.layer.cornerRadius = 20
+        continueButton.clipsToBounds = true
+        continueButton.setBackgroundColor(Colors.purple)
+        continueButton.titleLabel?.font = Fonts.AccountNotLogged.continue
```

Comment:
Look like this button configuration is repeat in `ForgetPasswordVC.swift` , can we have a global button ?